pkg/sql/catalog/tabledesc/validate.go

```diff
@@ -719,11 +702,12 @@ func (desc *wrapper) ValidateSelf(vea catalog.ValidationErrorAccumulator) {
 	// Only validate column families, constraints, and indexes if this is
 	// actually a table, not if it's just a view.
 	if desc.IsPhysicalTable() {
+		desc.validateConstraintNamesAndIDs(vea)
```

Comment:
Moving this further ahead was necessary because we got rid of `CheckUniqueConstraints`, but the price to pay was lots of noise in `validate_test.go`.